doc/README_Generic_Autotools.md

```diff
@@ -0,0 +1,50 @@
+Autotools Based YaST Packages
+=============================
+
+This is a generic documenation for YaST packages which use autotools (autoconf/automake)
+for building the package.

```

Comment:
How to find out that the project uses autoconf/automake? Special file in repo, directory?